src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -668,7 +668,7 @@ private static CultureData CreateCultureWithInvariantData()
         internal static CultureData? GetCultureData(string? cultureName, bool useUserOverride)
         {
             // First do a shortcut for Invariant
-            if (string.IsNullOrEmpty(cultureName))
+            if (string.IsNullOrEmpty(cultureName) || cultureName.Equals("und", StringComparison.OrdinalIgnoreCase))
```

Comment:
Should we include a constant here with more comments about what this is and why it's treated this way?

Reply:
Good idea!